### PR TITLE
Refactor/pcc/responsive

### PR DESCRIPTION
**Pull request by FredericEspiau**
MANAGER-4409

# Validation policy
It's been validated by two peers already, but I've rebase in the meantime

# Merging policy
Please don't squash, the commits should be good enough 🙏

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/index.js [hunk 1]
```diff
@@ -21,6 +22,11 @@ angular
     'ui.router',
   ])
   .component(component.name, component)
+  .config(
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/index.js]: Please use the same pattern as usually `.config(routing)`

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import component from './dedicatedCloud-dashboard.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicatedClouds.dashboard', {
+    component,
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js]: No need to use the angular declared component ?

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dedicatedCloud.html [hunk 3]
```diff
@@ -7,85 +7,74 @@
...
+            <div class="oui-page-header__container d-block">
+                <div class="oui-page-header__content">
+                    <div class="d-flex">
+                        <h1
+                            class="oui-page-header__title"
+                            data-ng-bind="dedicatedCloud.description"
+                        ></h1>
+                        <button
+                            type="button"
+                            class="btn btn-icon ml-2"
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dedicatedCloud.html]: `oui-button` class instead maybe ?

Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/dedicatedCloud.html]: Added some nasting padding that I couldn't get rid of unless I create a new stylesheet 😢

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js (hunk 2 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js]: Good point, I'll check

Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js]: It was unused so I removed the declaration ! Good find @marie-j

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dedicatedCloud-datacenter.html [hunk 4]
```diff
@@ -1,118 +1,118 @@
-<div class="dedicatedCloud-datacenter">
-    <div
-        class="text-center mt-5"
-        data-ng-if="$ctrl.loadingInformations && !$ctrl.loadingError"
-    >
-        <oui-spinner data-size="l"></oui-spinner>
-    </div>
+<div
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dedicatedCloud-datacenter.html]: Please don't look at this whole file, it's mostly whitespaces, you can suppress them by clicking on "Hide whitespace changes" 🙏

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import component from './dedicatedCloud-dashboard.component';
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routes.js]: We tends to name file that defined a route in something like: `foo.routing.js`.

What do you think about it?

In comparison:
```sh
$ find . -type f -name "*.routing.js" | wc -l
378

$ find . -type f -name "*.routes.js" | wc -l
106
```

**Pull request merged**